app/pillar/handler/crud.go

```diff
@@ -101,3 +101,53 @@ func DeleteSearch(c *web.AppContext) {
 	err := service.DeleteSearch(c)
 	doRespond(c, nil, err)
 }
+
+func CreateUpdateForm(c *web.AppContext) {
+	dbObject, err := service.CreateUpdateForm(c)
+	doRespond(c, dbObject, err)
+}
+
+func UpdateFormStatus(c *web.AppContext) {
+	dbObject, err := service.UpdateFormStatus(c)
+	doRespond(c, dbObject, err)
+}
+
+func GetForms(c *web.AppContext) {
```

Comment:
GetForms returns many forms, GetForm returns one by id.

You can see how the routes are formed here: https://github.com/coralproject/pillar/blob/ask-apis/app/pillar/route/routes.go#L51